src/lib/progress-spinner/progress-spinner.ts

```diff
@@ -28,28 +28,28 @@ const endIndeterminate = 80;
 /* Maximum angle for the arc. The angle can't be exactly 360, because the arc becomes hidden. */
 const MAX_ANGLE = 359.99 / 100;
 
-export type ProgressCircleMode = 'determinate' | 'indeterminate';
+export type ProgressSpinnerMode = 'determinate' | 'indeterminate';
 
 type EasingFn = (currentTime: number, startValue: number,
                  changeInValue: number, duration: number) => number
 
 
 /**
- * <md-progress-circle> component.
+ * <md-progress-spinner> component.
  */
 @Component({
   moduleId: module.id,
-  selector: 'md-progress-circle, mat-progress-circle',
+  selector: 'md-progress-spinner, mat-progress-spinner',
```

Comment:
We need to leave the old selectors as well so we can more easily migrate existing google projects